spec/requests/works_show_request_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+
+describe "Works#show", :type => :request do
+  let(:work) { create(:work) }
+  let(:chapter) { work.chapters.first }
+
+  context "when the first chapter of a work is unposted" do
+    before do
+      work.chapters.create(position: 1, posted: false, content: "Draft content")
+      chapter.update(position: 2)
+    end
+
+    it "displays the first posted chapter" do
```

Comment:
This is good for logged out users, but could we also cover

* when logged in as admin, it displays the first chapter regardless of posted status
* when logged in as work creator, it displays the first chapter regardless of posted status
* when logged in as an invited work creator, it displays the first chapter regardless of posted status
* when logged in as a random user, it displays the first posted chapter

Reply:
I don't know what any of the rspec tests are. They must be from the old person when I pulled their changes to work on this. I do tests exclusively using cucumber

Reply:
In that case, let's take all the Rspec tests out! But we do need all of those scenarios covered in Cucumber.

Reply:
Hi, @britnesissom! It's been a minute, so I wanted to check in on this. Do you plan to tackle the requested changes to the tests? If not, we can have someone else take over.